tools/secret-rotation/Azure.Sdk.Tools.SecretRotation.Tests/TestConfigurations/Valid/random-string.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "https://raw.githubusercontent.com/azure/azure-sdk-tools/main/schemas/secretrotation/1.0.0/schema.json",
+  "$schema": "https://raw.githubusercontent.com/azure/azure-sdk-tools/main/tools/secret-rotation/schema/1.0.0/plan.json",
```

Comment:
Is this the new desired pattern for schemas?

Reply:
Moving the schema to the tool folder was requested.

Reply:
Yes I wanted to keep the root of the tools repo clean and keep these scoped under the tool that uses them.